### DISCUSSION TITLE
Add a placeholder to order any statistic in a leaderboard

### DISCUSSION THREAD

**Discussion by Recliness** [Zrips/CMI#8129] [Feature Requests]
For example, if I wanted to make a kill leaderboard it would be cool if cmi had the built-in option to have placeholders that would allow me to do that, like %cmi_topstats_kills_[1-10+]%

**Comment by CloudeLecaw** [comment]
in the meantime until this might is added on day if it ever will be you can use a plugin like this, what does excatly what you suggested. (just the placeholdernames are bit different XD. It can create leaderbords from everything that has numerical placeholder value)
https://www.spigotmc.org/resources/ajleaderboards.85548/

**Reply by Recliness** [reply]
I was told this plugin has memory leaks

**Reply by CloudeLecaw** [reply]
well we also use it and didnt noticed memory leaks in it. maybe out server is just to powerfull xD
But you should give it a try, because i aspect it will take a much longer time before something like will be added.... because it would need something that can make tops from all stats that are saved in all playerfiles, what meant it takes also much source to read all player statisics to messure them..specially when you have over xxxxxx playerfiles.... so i aspect its a nogo (or better a will not be used) on bigger servers. stats are not written to cmi databse like playtime for baltops. etc.. they are saved in your playerfiles stats folder (in main world folder)

**Reply by CloudeLecaw** [reply]
if you dont like Ajleaderboards then what about this one?
https://www.spigotmc.org/resources/globalstats-minecraft-leaderboards-with-offline-external-plugin-support.103445/

**Comment by BenHeroPH** [comment]
is this implemented now?

**Reply by AllFiRE0** [reply]
owo
